server/routerlicious/packages/lambdas-driver/src/document-router/documentPartition.ts

```diff
@@ -15,7 +15,6 @@ import {
 } from "@fluidframework/server-services-core";
 import { getLumberBaseProperties, Lumberjack } from "@fluidframework/server-services-telemetry";
 import { QueueObject, queue } from "async";
-import * as _ from "lodash";
```

Comment:
Weird! This is likely old code that was never cleaned up properly, but does lodash have side effects that execute on import? @znewton @hedasilv Not sure if this is your area of expertise, but since this is server code, want to make sure we're not inadvertently breaking something. Do you know if this import is relevant?

Reply:
Thanks for the ping, Tyler! It's not my area of expertise, but I checked everything I could and it looks totally safe to remove the 2 "lodash" imports being removed here (this file and kafka-service/src/partition).

Reply:
I also pinged @hedasilv to see if he knows about some weird global artifact from this import that is important

Reply:
I agree with Zach, but just to be safe, we could try to run r11s locally in Docker and target it with a test app like clicker. Let me try to test that some time today and hopefully soon I can get back to you.

Reply:
Actually, we might not even have to run r11s on Docker with Clicker. Looking into the code history, those imports of `lodash` were added so that we could use `_.cloneDeep()` as in here: https://github.com/microsoft/FluidFramework/commit/09e34b1942fe32792590cfc0062818615a26ebed and here: https://github.com/microsoft/FluidFramework/pull/1606/files

That code was added to partition.ts and documentPartition.ts a few years ago (2 and 4.5, specifically) and those files do not do `_.cloneDeep()`. So I think it's safe to remove lodash.

I will give the approval from the server side 😄

Reply:
THanks, gentlemen! @Josmithr Merge away!